packages/native-extensions/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es2019",
+    "baseUrl": "./src",
+    "outDir": "./dist",
+    "typeRoots": ["../../node_modules/@types"],
+    "types": ["@types/node"]
+  }
+}
```

Comment:
Instead of copy-pasting this file everywhere, we could consider having a root tsconfig that we reference in each projects.

Reply:
Yup, i'll add this to the bucket list ! Some options are used everywhere(eg: strict, interop, etc) but we havent tested module es2019 / es2021 everywhere, so we still rely on individual files. It'll come in another pr :)